--- DateTime-Format-Mail-0.30/Build.PL.orig	2007-03-25 02:29:49.000000000 +0000
+++ DateTime-Format-Mail-0.30/Build.PL	2007-03-25 02:31:08.000000000 +0000
@@ -5,7 +5,7 @@
 Module::Build->new( module_name => 'DateTime::Format::Mail',
                     author      => 'Dave Rolsky <autarch@urth.org>',
                     license     => 'perl',
-                    requires    => { 'DateTime' => '0.1705',
+                    requires    => { 'DateTime' => '0.17.05',
                                      'Params::Validate' => '0.67',
                                      'Test::More' => '0.47',
                                    },
--- DateTime-Format-Mail-0.30/META.yml.orig	2007-03-25 02:29:58.000000000 +0000
+++ DateTime-Format-Mail-0.30/META.yml	2007-03-25 02:30:48.000000000 +0000
@@ -11,7 +11,7 @@
 resources:
   license: http://dev.perl.org/licenses/
 requires:
-  DateTime: 0.1705
+  DateTime: 0.17.05
   Params::Validate: 0.67
   Test::More: 0.47
 build_requires:
--- DateTime-Format-Mail-0.30/lib/DateTime/Format/Mail.pm.orig	2007-03-25 02:30:06.000000000 +0000
+++ DateTime-Format-Mail-0.30/lib/DateTime/Format/Mail.pm	2007-03-25 02:30:59.000000000 +0000
@@ -4,7 +4,7 @@
 use strict;
 use 5.005;
 use Carp;
-use DateTime 0.1705;
+use DateTime 0.17.05;
 use Params::Validate qw( validate validate_pos SCALAR );
 use vars qw( $VERSION );
 
